### PR TITLE
LUD-1629 increase timeout for FW update

### PR DESCRIPTION
**Pull request by danielseankim**

### CODE REVIEW

**Comment by danielseankim** [comment]
@gavin-scott please review & merge

File: lib/puppet/provider/idrac_fw_update/wsman.rb [hunk 1]
```diff
@@ -213,7 +213,7 @@ def get_data
  
   def clear_job_queue
     Puppet.debug("Clearing Job Queue")
-    wsman_cmd = "wsman invoke -a \"DeleteJobQueue\" http://schemas.dell.com/wbem/wscim/1/cim-schema/2/DCIM_JobService?CreationClassName=\"DCIM_JobService\"&SystemName=\"Idrac\"&Name=\"JobService\"&SystemCreationClassName=\"DCIM_ComputerSystem\" -N root/dcim -u #{transport[:user]} -p #{transport[:password]} -h #{transport[:host]} -P 443 -v -j utf-8 -y basic -o -m 256 -c Dummy -V  -k \"JobID=JID_CLEARALL\" "
+    wsman_cmd = "wsman invoke -a \"DeleteJobQueue\" http://schemas.dell.com/wbem/wscim/1/cim-schema/2/DCIM_JobService?CreationClassName=\"DCIM_JobService\"&SystemName=\"Idrac\"&Name=\"JobService\"&SystemCreationClassName=\"DCIM_ComputerSystem\" -N root/dcim -u #{transport[:user]} -p #{transport[:password]} -h #{transport[:host]} -P 443 -v -j utf-8 -y basic -o -m 256 -c Dummy -V  -k \"JobID=JID_CLEARALL_FORCE\" "
```
Comment (gavin-scott) [inline lib/puppet/provider/idrac_fw_update/wsman.rb]: Why was this added?

Comment (danielseankim) [reply lib/puppet/provider/idrac_fw_update/wsman.rb]: for clearing job. Since we switched back and forth for using or not using puppet module, this was lost in the crack.
